### DISCUSSION TITLE
Unable to install tailwindCSS because a folder does not exist

### DISCUSSION THREAD

**Discussion by olivepierre** [tailwindlabs/tailwindcss#10536] [Help]
Hi everyone,

I can't install "tailwindCSS" using pnpm because the folder "oxide-node-api-shim" does not exist.

As soon as I add this folder in my project, I can install tailwindCSS.

My Node version is 19.5
My OS is Windows.
My bundler is Vite.

You can find below my package.json :

``
{
  "name": "frontend",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "format": "prettier . --write",
    "preview": "vite preview",
    "test:unit": "vitest watch",
    "lint:scripts": "eslint --ext .ts,.tsx src --fix --cache --cache-strategy metadata --max-warnings 0",
    "storybook": "storybook dev -p 6006",
    "build-storybook": "storybook build"
  },
  "dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@storybook/addon-essentials": "7.0.0-beta.41",
    "@storybook/addon-interactions": "7.0.0-beta.41",
    "@storybook/addon-links": "7.0.0-beta.41",
    "@storybook/blocks": "7.0.0-beta.41",
    "@storybook/react": "7.0.0-beta.41",
    "@storybook/react-vite": "7.0.0-beta.41",
    "@storybook/testing-library": "0.0.14-next.1",
    "@testing-library/dom": "^8.20.0",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^14.4.3",
    "@types/react": "^18.0.27",
    "@types/react-dom": "^18.0.10",
    "@typescript-eslint/eslint-plugin": "^5.51.0",
    "@typescript-eslint/parser": "^5.51.0",
    "@vitejs/plugin-react": "^3.1.0",
    "autoprefixer": "^10.4.13",
    "eslint": "^8.33.0",
    "eslint-config-prettier": "^8.6.0",
    "eslint-config-standard-with-typescript": "^34.0.0",
    "eslint-plugin-eslint-comments": "^3.2.0",
    "eslint-plugin-import": "^2.27.5",
    "eslint-plugin-jsx-a11y": "^6.6.1",
    "eslint-plugin-n": "^15.6.1",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-promise": "^6.1.1",
    "eslint-plugin-react": "^7.32.2",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-storybook": "^0.6.10",
    "jsdom": "^21.1.0",
    "msw": "^1.0.0",
    "postcss": "^8.4.21",
    "prettier": "2.8.4",
    "prop-types": "15.8.1",
    "storybook": "7.0.0-beta.41",
    "tailwindcss": "^3.2.5",
    "typescript": "^4.9.5",
    "vite": "^4.1.0",
    "vitest": "^0.28.4"
  }
}
``

Looks like a bug to me.

It is not blocking at all, but quite annoying, as I need to keep an empty folder to be able to install tailwindCSS :-(

Thanks for your help !

**Comment by Stephen-Almendinger** [comment]
I am receiving this same error message when trying to install tailwind.

**Reply by olivepierre** [reply]
I am trying to replicate with npm (instead of pnpm) but I have the same message : npm is unable to find 'node_modules\tailwindcss\oxide-node-api-shim\package.json'

It does not seem related to npm/pnpm !

**Reply by olivepierre** [reply]
Just saw it in the issues: 
https://github.com/tailwindlabs/tailwindcss/issues/10534

Right now, it is recommended to install 3.2.4 (or the latest npm).

Thanks Tailwind for your great work !
